src/app/clusters/operational-state-server/operational-state-server.cpp

```diff
@@ -399,6 +453,8 @@ void Instance::HandleStopState(HandlerContext & ctx, const Commands::Stop::Decod
     response.commandResponseState = err;
 
     ctx.mCommandHandler.AddResponse(ctx.mRequestPath, response);
+
+    UpdateCountdownTimeFromClusterLogic();
```

Comment:
I think this should not be here either, at least not if `opState == to_underlying(OperationalStateEnum::kStopped)` so we never call into the delegate and never change state....

And if we _do_ call into the delegate and change state, won't that trigger this reporting anyway?

Reply:
Done